PureLayout/PureLayout/PureLayoutDefines.h

```diff
@@ -34,10 +34,18 @@
 #define __PureLayout_MinBaseSDK_iOS_8_0                   (TARGET_OS_IPHONE && __IPHONE_OS_VERSION_MAX_ALLOWED > __IPHONE_7_1)
 #define __PureLayout_MinBaseSDK_OSX_10_10                 (!TARGET_OS_IPHONE && __MAC_OS_X_VERSION_MAX_ALLOWED > __MAC_10_9)
 
+#ifdef __MAC_10_11
+#define __PureLayout_MinBaseSDK_OSX_10_11                 (!TARGET_OS_IPHONE && __MAC_OS_X_VERSION_MAX_ALLOWED >= __MAC_10_11)
```

Comment:
I thought about using `__MAC_OS_X_VERSION_MAX_ALLOWED > __MAC_10_10_3`, but the tricky thing is that we don't know if there will be a new minor version for 10.10 this year.